.github/workflows/e2e.yaml

```diff
@@ -11,8 +11,34 @@ env:
   NAMESPACE: grafana-operator-system
 
 jobs:
+  docs_only_check:
```

Comment:
Create a check to not run e2e test if there is only documentation changes.

Reply:
This won't work during this PR due to a few files is .yaml. And I do want to e2e test yaml changes.